data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/beacon/GetStateFork.java

```diff
@@ -71,8 +71,7 @@ public GetStateFork(final DataProvider dataProvider, final JsonProvider jsonProv
       method = HttpMethod.GET,
       summary = "Get state fork",
       tags = {TAG_V1_BEACON, TAG_VALIDATOR_REQUIRED},
-      description =
-          "Request beacon node to provide all validators that are scheduled to propose a block in the given epoch.",
+      description = "Returns Fork object for state with given 'stateId'.",
```

Comment:
`state_id`

Reply:
Fixed in the other PR, which this builds upon. Hoping to merge that one first.